packages/livekit-server-sdk/src/digest.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2024 LiveKit, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Use the Web Crypto API if available, otherwise fallback to Node.js crypto
+export default async function digest(algorithm: string, data: ArrayBuffer): Promise<ArrayBuffer> {
+  if (globalThis.crypto?.subtle) {
+    return globalThis.crypto.subtle.digest(algorithm, data);
+  } else {
+    const nodeCrypto = await import('node:crypto');
+    const hash = nodeCrypto.createHash(algorithm.replace('-', '').toLowerCase());
```

Comment:
I don't believe we're likely to change the algorithm here in the foreseeable future. I'd rather go with hard coded algorithm names here and above instead of the string manipulation.